subprojects/core/src/integTest/groovy/org/gradle/api/AntBuilderLoggingIntegrationTest.groovy

```diff
@@ -21,7 +21,7 @@ import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 class AntBuilderLoggingIntegrationTest extends AbstractIntegrationSpec {
     def setup() {
         executer.requireIsolatedDaemons()
-        buildFile << """
+        buildFile """
```

Comment:
:question: Can't we keep the leftshift operator?  Previously the semantic was to append to the file. What's the new semantic?

Reply:
The left shift operator is on the type `TestFile` which could be a fike of any kind. We'd have to return a subtype of `TestFile` with the left shift operator properly annotated for that to work most likely. I was less keen on that change.

What's happening now is `buildScript(@GroovyBuildScriptLanguage String contents)` is invoked

Reply:
I see. Thanks for clarifying

Reply:
I think there should be a set of more primitive methods that only do the highlighting so there's less risk of accidentally changing semantics when using them, for example:

```groovy
buildFile << groovyBuildscript("....") // Build script highlighting
settingsFile << groovySettings("...") // Settings script highlighting
initFile << groovyInitScript("...") // Init script highlighting
```

Reply:
This seems overly verbose to me.

```groovy
buildFile """
// Congrats you get syntax highlighting
"""
```
vs
```groovy
buildFile << """
// No syntax highlighting for you
"""
```

I don't see any down sides to adding the `groovyBuildscript`, `groovySettings`, `groovyInitScript` methods in addition, but I like that `buildFile """...` is kinda like syntactic sugar simplifying the call.

Reply:
I think it is fine to keep the `buildFile` for now and add/refine things later.